src/UpupUploader.tsx

```diff
@@ -36,6 +50,7 @@ export const UpupUploader: FC<UpupUploaderProps> = ({
             bucket={bucket}
             setKey={setKey}
             canUpload={canUpload}
+            compressible={compressible}
```

Comment:
Maybe `compress` makes more sense than `compressible` ?

Reply:
the best practices says that the  name of boolean has a prifix with: is/has/can + noun like isCompressed , hasAge, canUpload OR simply the adjective like sorted, hidden , compressible

Reply:
i will change it to shouldBeCompressed on the next release

Reply:
We settled on `toBeCompressed`